taxi/models.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from taxi_service import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True)
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE)
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL)

```

Comment:
pls add related names to ur Car model) Dont forget about [checklist.md](https://github.com/mate-academy/py-taxi-service-startproject/blob/master/checklist.md)